airflow/models/session.py

```diff
@@ -0,0 +1,52 @@
+# Licensed to the Apache Software Foundation (ASF) under one
+# or more contributor license agreements.  See the NOTICE file
+# distributed with this work for additional information
+# regarding copyright ownership.  The ASF licenses this file
+# to you under the Apache License, Version 2.0 (the
+# "License"); you may not use this file except in compliance
+# with the License.  You may obtain a copy of the License at
+#
+#   http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing,
+# software distributed under the License is distributed on an
+# "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+# KIND, either express or implied.  See the License for the
+# specific language governing permissions and limitations
+# under the License.
+
+# ~~~~~~~~~~~~~~~~~~~~~~
+#
+# Server-side Sessions and SessionInterfaces.
+#
+# Originally copied from flask_session.sessions
+# Flask-Session
+# Copyright (C) 2014 by Shipeng Feng and other contributors
+# BSD, see LICENSE for more details.
+from datetime import datetime
+from typing import Any, Dict
+
+from sqlalchemy import Column, DateTime, Integer, LargeBinary, String
+
+from airflow.models.base import Base
+from airflow.utils.log.logging_mixin import LoggingMixin
+
+
+# This simply matches the model in `fask_session.sessions.SqlAlchemySessionInterface`
+class Session(Base, LoggingMixin):
```

Comment:
I'm not sure if the model itself is enough to warrant keeping the copyright notice and addition to NOTICE. Thoughts?

Reply:
it's a good question, firmly outside my bailywick, but intuition says _no_

Reply:
it seems to me that a simple database table definition, if that is all that is common, is not meaningful enough

Reply:
I would err on the side of caution and keep the attribution.

Reply:
No longer an issue 🎉